make-dist

```diff
@@ -68,6 +76,25 @@ download_boost() {
     rm -rf src/boost
 }
 
+download_liburing() {
+    liburing_version=$1
+    shift
+    liburing_sha256=$1
+    shift
+    liburing_fname=liburing-${liburing_version}.tar.gz
+    download_from $liburing_fname $liburing_sha256 $*
+    tar xzf $liburing_fname -C src  \
```

Comment:
On SUSE, I think this `tar xzf` does require the package `gzip` to be installed in the build environment.